### PR TITLE
feat: implement 0x11 - GT opcode

### PR DESCRIPTION
**Pull request by dbejarano820**
## Pull Request type
Please check the type of change your PR introduces:

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
Not implemented

Resolves: #6 

## What is the new behavior?
Implements opcode

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by dbejarano820** [comment]
@enitrat thoughts on test function names?

File: crates/evm/src/instructions/comparison_operations.cairo [hunk 1]
```diff
@@ -20,10 +20,19 @@ impl ComparisonAndBitwiseOperations of ComparisonAndBitwiseOperationsTrait {
     /// 0x11 - GT
     /// # Specification: https://www.evm.codes/#11?fork=shanghai
     fn exec_gt(ref self: ExecutionContext) -> Result<(), EVMError> {
-        Result::Ok(())
+        let popped = self.stack.pop_n(2)?;
+        let a = *popped[0];
+        let b = *popped[1];
+        let result = a > b;
```
Comment (enitrat) [inline crates/evm/src/instructions/comparison_operations.cairo:26]: to keep it consistent with other opcodes i would prefer something like 

```rust
    let result = if (a > b) {
        1
    } else {
        0
    };
    ```

Comment (enitrat) [reply crates/evm/src/instructions/comparison_operations.cairo:26]: and then directly stack.push the result 😄

File: crates/evm/src/tests/test_instructions/test_comparison_operations.cairo [hunk 2]
```diff
@@ -196,3 +196,67 @@ fn test_byte_offset_out_of_range() {
     assert(ctx.stack.len() == 1, 'stack should have one element');
     assert(ctx.stack.peek().unwrap() == 0x00, 'stack top should be 0x00');
 }
+
+#[test]
+#[available_gas(20000000)]
+fn test_greater_than() {
```
Comment (enitrat) [inline crates/evm/src/tests/test_instructions/test_comparison_operations.cairo:202]: not really useful given that we have all cases covered afterwards

Comment (enitrat) [reply crates/evm/src/tests/test_instructions/test_comparison_operations.cairo:202]: can you name them `test_exec_<opcode>_...` ?

**Review by enitrat — DISMISSED** [review]
lgtm 💪

**Comment by enitrat** [comment]
format failed @dbejarano820 you will need to run scarb fmt

**Comment by dbejarano820** [comment]
done! ty boss @enitrat

File: crates/evm/src/instructions/comparison_operations.cairo [hunk 3]
```diff
@@ -20,7 +20,16 @@ impl ComparisonAndBitwiseOperations of ComparisonAndBitwiseOperationsTrait {
...
```
Comment (enitrat) [inline crates/evm/src/instructions/comparison_operations.cairo:26]: You still have an extra line here

Comment (dbejarano820) [reply crates/evm/src/instructions/comparison_operations.cairo:26]: damn

**Review by enitrat — APPROVED** [review]
Lfg🦅🇺🇸

**Review by enitrat — APPROVED** [review]
LGTM!

**Pull request merged**